scripts/services/CreateConflationTypeGroups.py

```diff
@@ -81,7 +81,60 @@ def createUiJSON(groups, options):
-                        if len(templateDefault) == 1:
+                        # If default points to file of values
+                        if 'HOOT_HOME' in memberConfig['default']:
+                            filePath = os.path.expandvars(memberConfig['default'])
+                            if os.path.isfile(filePath):
+                                with open(filePath, 'r') as jsonFile:
+                                    data = json.load(jsonFile)
+                                    for (key, value) in data.items():
+                                        #Use key as label for what the objects inside represent
+                                        category = key.replace('_', ' ').capitalize()
+
+                                        # ignore comments in the json
+                                        if key != '#' and isinstance(value, list):
+                                            for categoryObj in value:
+                                                memberObject = {
+                                                    'id': match,
+                                                    'input': 'text',
+                                                    'type': 'json',
+                                                    'parentKey': key
+                                                }
+                                                objKeys = []
+
+                                                if 'key' in categoryObj:
+                                                    categoryKey = categoryObj['key']
+                                                    objKeys.append('key')
+                                                    memberObject['key'] = categoryKey
+                                                else:
+                                                    categoryKey = ''
+
+                                                if 'value' in categoryObj:
+                                                    categoryVal = categoryObj['value']
+                                                    objKeys.append('value')
+                                                    memberObject['value'] = categoryVal
+                                                else:
+                                                    categoryVal = '*'
+                                               
+                                                if 'distance' in categoryObj:
+                                                    categoryDefault = categoryObj['distance']
+                                                    objKeys.append('distance')
+                                                    memberObject['distance'] = categoryDefault
+                                                else:
+                                                    categoryDefault = ''
+
+                                                label = category + ' [' + categoryKey + '=' + categoryVal + ']'
```

Comment:
I think we could use a `(m)` unit here, but that might make it even less generic.

Reply:
that was my concern as well